.github/workflows/env_prod.yml

```diff
@@ -1846,7 +1846,8 @@ jobs:
               REDIS_PORT="$redis_sslPort" \
               REDIS_KEY="$redis_key" \
               REFERENCE_DATA_PROXY_URL="http://$reference_data_proxy_url" \
-              HTTPS=1
+              HTTPS=1 \
+              START_PAGE_REDIRECT=1
```

Comment:
this would re-deploy the prod infrastructure if we merge it in. 
I'd like to avoid that unless it's really necessary. 
Can we change the triggers for this deployments though to a different branch? `infrastructure-prod` or something like that